lib/location/method_gnss.c

```diff
@@ -72,8 +72,12 @@ static K_SEM_DEFINE(entered_rrc_idle, 0, 1);
 #if (defined(CONFIG_NRF_CLOUD_AGPS) || defined(CONFIG_NRF_CLOUD_PGPS))
 static struct nrf_modem_gnss_agps_data_frame agps_request;
 #if !defined(CONFIG_NRF_CLOUD_MQTT)
+#if defined(CONFIG_NRF_CLOUD_REST_FRAGMENT_SIZE)
```

Comment:
Should we just use accept the use of a magic number 1700 and drop the if defined -clause? It would simplify the code. Or another option would be to create a new macro the value of which would depend on whether CONFIG_NRF_CLOUD_REST_FRAGMENT_SIZE is defined or not.
Same comment to CONFIG_NRF_CLOUD_REST_RECV_TIMEOUT below.

Reply:
These are removed now after other flagging changes